index.md

```diff
@@ -121,6 +121,12 @@ To achieve the best results you should have an example application like 'bookinf
 
 Kiali is open source and released under [Apache License v2](https://www.apache.org/licenses/LICENSE-2.0.txt)
 
+## Reporting issues and feature requests
+
+The Kiali team uses [JIRA at JBoss.org](http://issues.jboss.org/browse/KIALI) as its preferred tracker. If you have an account there, please use it to report issues and create new feature requests.
```

Comment:
perhaps worth to clarify that "report issues" means creating issues with `bug` type?